board/models.py

```diff
@@ -1,3 +1,96 @@
+from django.contrib.auth.models import User
 from django.db import models
 
 # Create your models here.
+
+
+class Board(models.Model):
+    """
+    A database model representing a board, where posts can be created and viewed.
+
+    This model uses a `BinaryField` as opposed to an `ImageField` to store the background image of
+    the board. While this is considered poor design in many cases, Heroku (the cloud provider
+    to be used)'s ephemeral file system renders the file system impossible to use for dynamic
+    storage. The is an issue because `ImageField`, which is a subclass of `FileField`, defaults to
+    storing files on the file system under `MEDIA_ROOT`.
+
+    As a workaround, this model stores the image as a BLOB in the actual database. While the
+    database may inflate as a result, the preservation of the images is guaranteed.
+
+    This board stores a many-to-many relationship to `django.contrib.auth.models.User`, the
+    default `User` model generated by Django. Since Django offers a `ManyToManyField` as a field,
+    an explicit lookup table is not necessary. Instead, users can be added to a board's
+    `admin_users` field, and Django can use lookups across relationships to find all admin users for
+    one specific board. Provided is an example:
+
+    ```
+    # generate and save mock users and boards
+    >>> u1 = User(username="shari", password="i love you") 
```

Comment:
Can you mention that `User` is a built-in model

Reply:
That is mentioned above when I specify `django.contrib.auth.models.User`, but I can add another note.

Reply:
just add a quick note, something like: `// the User from django`